homeworks/sviatoslav.makhynko_SvMak/homework_2/app.js

```diff
@@ -0,0 +1,70 @@
+// Initial state
+let state = {
+    height: 4,
+    spaces: 2,
+    symbol: '#',
+}
+
+// Getting arguments from the console
+process.argv.forEach(arg => {
+    arg = arg.split('=');
+
+    switch (arg[0]) {
+        case 'height':
+            return state.height = parseInt(arg[1], 10);
+        case 'spaces':
+            return state.spaces = parseInt(arg[1], 10);
+        case 'symbol':
+            return state.symbol = arg[1];
+    }
+});
+
+// Prints pyramids
+const piramidPrints = state => {
+    const { height } = state;
+
+    let str = `Height: ${height}\n`;
+
+    for (let i = 1; i <= height; i++) {
+        str += lineCreate(i, state)+'\n';
+    }
+
+    console.log(str);
+}
+
+// Creates spaces between pyramids
+function spacesCreate(spaces) {
```

Comment:
hmm, in function spacesCreate we get spaces as a parameter. It seems more useful name for parameter would be amount